src/Ship.cpp

```diff
@@ -379,50 +379,55 @@ inline int sign(T num)
 vector3d Ship::CalcAtmosphericForce() const
 {
 	// Data from ship.
-	double m_topCrossSec = GetShipType()->topCrossSection;
-	double m_sideCrossSec = GetShipType()->sideCrossSection;
-	double m_frontCrossSec = GetShipType()->frontCrossSection;
+	const auto topCrossSec = GetShipType()->topCrossSection;
+	const auto sideCrossSec = GetShipType()->sideCrossSection;
+	const auto frontCrossSec = GetShipType()->frontCrossSection;
 
 	// TODO: vary drag coefficient based on Reynolds number, specifically by
 	// atmospheric composition (viscosity) and airspeed (mach number).
-	double m_topDragCoeff = GetShipType()->topDragCoeff;
-	double m_sideDragCoeff = GetShipType()->sideDragCoeff;
-	double m_frontDragCoeff = GetShipType()->frontDragCoeff;
+	const auto topDragCoeff = GetShipType()->topDragCoeff;
+	const auto sideDragCoeff = GetShipType()->sideDragCoeff;
+	const auto frontDragCoeff = GetShipType()->frontDragCoeff;
 
-	double m_shipLiftCoeff = GetShipType()->shipLiftCoefficient;
+	const auto shipLiftCoeff = GetShipType()->shipLiftCoefficient;
```

Comment:
This is partially my bad for setting a bad example originally, but I'd prefer that we restrict the use of `auto` to iterator declarations and statements where the full type of the variable is declared as a template parameter on the right hand side - e.g. `reinterpret_cast<T>` or `LuaPull<T>`. It helps to keep the code readable and sane, and something that I'm personally trying to check for when submitting my own PRs now.

Reply:
Clear. I only wish to leave **const** specifier

Reply:
No problem with that! I'd just prefer we not have `auto` infect our code until we have to run static analysis on it to figure out what the type of a variable actually *is* :smile: